Source/Plugin.LocalNotification/Platform/iOS/NotificationServiceImpl.cs

```diff
@@ -96,8 +99,18 @@ public async Task<bool> Show(NotificationRequest request)
                     return false;
                 }
 
-                var allowed = await NotificationCenter.AskPermissionAsync().ConfigureAwait(false);
-                if (allowed == false)
+                if(FilterNotification != null)
```

Comment:
You cannot do the filter here, have to do it in LocalNotificationDelegate.WillPresentNotification